### ISSUE TITLE
Pages for workshops

### ISSUE THREAD

**Issue by kdmnk** [luksan47/mars#421]
We can make pages for the workshops just like on our public site (https://eotvos.elte.hu/szakmai-munka/muhelyek) but here we could update members (and courses, later) dinamically.

However, I don't know if we want a section of Urán to be publicly available. Also we would need to update our gdpr statement for this.

**Comment by luksan47** [comment]
I don't think we should use Uran as a public tool. It might worth considering if we could migrate some data to the actual website of the Collegium, that would be useful.

**Comment by kdmnk** [comment]
If we could generate the html code that we can paste into the public website, that would be good as well.
This way, we can unifiy the pages easily too.

**Comment by kdmnk** [comment]
I am thinking of a page where the workshop leaders can easily add courses, competition results, etc.,  and then from these data we generate the html file (including the automatically generated member list) that we can just paste into our website.  @luksan47 what do you think?

**Comment by luksan47** [comment]
I have the basics for adding courses in a PR,  the others would be great as well. If we could generate the html, and get notifications when there are updates, we could upload the generated files. Sounds good.

**Comment by kdmnk** [comment]
Actually, I was able to include an Uran page in the website in an <iframe>, but it seems tricky.
I'm wandering if we could request some json data and then include that with js....

Another point for this issue: we could have the workshop administrators update the website.
